content/en/docs/concepts/windows/intro.md

```diff
@@ -50,7 +50,8 @@ Some node features are only available if you use a specific
 including:
 
 * HugePages: not supported for Windows containers
-* Privileged containers: not supported for Windows containers
+* Privileged containers: not supported for Windows containers.
+  [HostProcess Containers](/docs/tasks/configure-pod-container/create-hostprocess-pod.md) offer similar functionality.
```

Comment:
```suggestion
  [HostProcess Containers](/docs/tasks/configure-pod-container/create-hostprocess-pod/) offer similar functionality.
```

---
_aside_
Ideally, we'd add a heading about HostProcess containers into the https://kubernetes.io/docs/concepts/containers/ section overview. This link could then be to that heading.

However, that might first mean adding a section into https://kubernetes.io/docs/concepts/containers/  about supported OSs for containers, which I think is best done in its own PR.